### DISCUSSION TITLE
Persian (Farsi) Translation

### DISCUSSION THREAD

**Discussion by saman-taghavi** [vuejs-translations/guidelines#6] [General]
I would like to start the persian translation, of course after the doc and translation Guidelines release. If anyone is interested or has some tips to point out, It would be Awesome.

**Comment by mehdinourollah** [comment]
I would love to help on the Persian Translation

**Reply by saman-taghavi** [reply]
Awsome

**Comment by psamipour** [comment]
count me in! 😄

**Comment by omidbakhshi** [comment]
I would help you too

**Comment by Muhamad1375** [comment]
I would like to help

**Comment by erfuun** [comment]
we can do it 😁

**Comment by amiria703** [comment]
Fine, I want to help too!

**Comment by mostafa-nematpour** [comment]
I would like to contribute

**Comment by alireza-keshavarz-d** [comment]
I can help with non-heavy-vue-related stuff.
i dont use vue much
but i can translate basic stuff like tutorials

**Comment by ARZarkesh** [comment]
I would like to contribute for generating Persian docs

**Comment by the-pesar** [comment]
I want to help too, who wants to start ?

**Reply by the-pesar** [reply]
Unfortunately, nobody started.
So I forked the Vue.js docs and am currently working on it,
help me for Persian(Farsi) translation please.
Repo: https://github.com/the-pesar/docs-fa
Telegram Group: [t.me/Vue_Docs_Fa](https://t.me/Vue_Docs_Fa)

**Reply by the-pesar** [reply]
@farad-tech @ARZarkesh @alumen2101 @mostafa-nematpour @amiria703 @erfuun @Muhamad1375 @omidbakhshi @psamipour @saman-taghavi @mehdinourollah

**Comment by farad-tech** [comment]
Hi everyone, I would like to cooperate 😍.

**Comment by alirezahamid** [comment]
Hey @Jinjiang @NataliaTepluhina @yyx990803,

Hope you're doing great! 😊

Just wanted to share some exciting news with you - we've begun translating the Vue docs into Farsi (Persian) and have made some great progress so far. We believe it would be fantastic if we could have a separate repository for fa.vuejs.org. It would not only boost our motivation but also encourage more translators to join us.

Oh, and guess what? Once we wrap up with the Vue.js docs, we're eager to dive into translating Nuxt and Pinia next!

Thanks for your support and looking forward to your thoughts.

Best wishes! 🌟

**Reply by alirezahamid** [reply]
@mostafa-nematpour @the-pesar

**Reply by Jinjiang** [reply]
@alirezahamid glad to know that! Great job!

According to the guidelines, please make sure:
1. you have completed translation for more than 50% of the content
2. you have [detached your fork](https://support.github.com/request/fork)

For more information: https://github.com/vuejs-translations/guidelines#high-level-steps-of-working-on-a-new-translation

Then we will help you to transfer the repo and create the official deployment. Thanks.

**Reply by the-pesar** [reply]
@Jinjiang 

1. We have successfully translated almost **50%** of the documentation content.
2. I also detached the [fork](https://github.com/the-pesar/docs-fa)

Please help me to transfer the repo and create the official deployment. Thanks.

**Reply by Jinjiang** [reply]
@the-pesar great job! I have sent to you an inviation to proceed. Thanks.

**Reply by the-pesar** [reply]
Thanks, I transferred.
Now how can i deploy that on [fa.vuejs.org](https://fa.vuejs.org) ?
@Jinjiang

**Reply by Jinjiang** [reply]
Cool. For the deployment we will take care of it and let you know once it's done. No worries. :-)

**Comment by mostafa-nematpour** [comment]
> @the-pesar great job! I have sent to you an inviation to proceed. Thanks.

Hello,
I am part of the Persian translation project. Is it possible for me to have access and be invited as well?

@Jinjiang

**Reply by Jinjiang** [reply]
sure. invitation sent

**Reply by mostafa-nematpour** [reply]
Thanks

To configure Ryu-cho, I need access to the project settings. Do I need to obtain the Maintainer role, or is there another action required?

**Reply by Jinjiang** [reply]
Done. You are maintainer now. Feel free to proceed.

**Reply by mostafa-nematpour** [reply]
Thank you. It seems like the project hasn't been added to our team. Please add `vuejs-translations/docs-fa` to the 'fa' team.
@Jinjiang

**Reply by Jinjiang** [reply]
Sure. I have granted all the team members as the admin role.

**Comment by yyx990803** [comment]
fa.vuejs.org is live :)

Congrats and thanks for the hard work.

**Reply by mostafa-nematpour** [reply]
Thanks 🤩

**Comment by the-pesar** [comment]
@yyx990803 @Jinjiang 

Greetings,

We are nearing completion of the Vue documentation translation into Persian. I wanted to express my sincere appreciation for your assistance thus far. In Iran, there is a sizable community of Vue developers who struggle with English fluency. Our translation efforts greatly aid this group, as evidenced by the 260 stars our repository accrued in just one week.
Emboldened by our success, we have decided to expand our translation work to cover Pinia and Vue Router. If agreeable, I humbly request that you grant our team an official repository within vuejs-translations. An official mandate would empower our translation process and lend further credibility. I genuinely believe our work stands to meaningfully help Persian-speaking developers engage with the Vue ecosystem. Please let me know if any additional information about our team or translation approach would help inform your decision. I look forward to your response.

Best regards, Yasha

**Reply by Jinjiang** [reply]
@the-pesar nice job! For translations of other official libs, we are discussing a simpler workflow (see https://github.com/vuejs/router/pull/1855 for more details). Once it's done, you are free to do the translations for like Pinia, Vue Router, etc. Thanks.

**Reply by the-pesar** [reply]
@Jinjiang Thank you, This is a good idea, however implementing it is not feasible at this time as the Persian language is written from right-to-left(RTL) and requires customization of the font and website template. What is your solution ???

**Reply by Jinjiang** [reply]
From what I tested so far, `dir="auto"` attribute is very helpful in general. We are discussing whether we can necessarily make it as the default setting for the VitePress theme.

At the same time, feel free to let me know any points you think we can customize so we can discuss further. And a demo git branch for that is also very welcome.

/cc @brc-dd

**Reply by Jinjiang** [reply]
@the-pesar a small update: we just announced a new guidelines for Vue ecosystem projects. Feel free to take a look: https://github.com/vuejs-translations/guidelines/blob/main/README_ECOSYSTEM.md.
